frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -25,6 +25,8 @@ const ManageCows = ({ userCommons, commons, onBuy, onSell }) => {
       <Card.Body>
         <Card.Title>Market Cow Price: ${commons?.cowPrice}</Card.Title>
         <Card.Title>Number of Cows: {userCommons.numOfCows}</Card.Title>
+        <Card.Title>Lifetime Cows Bought: {userCommons.lifetimeCowsBought}</Card.Title>
+        <Card.Title>Lifetime Cows Sold: {userCommons.lifetimeCowsSold}</Card.Title>
```

Comment:
Technically this will error if `userCommons` is null, but the the title above doesn't handle this case either so this PR doesn't add any new source of error. It just doubles down on this bad precedent for future contributors, but for time sake this should be merge-able.

Reply:
Could handle that with ?. as we did for commons a few lines above.   On the other hand, if we don't have a userCommons object, then we should not even be on this page.